src/interfaces/chat-bot.ts

```diff
@@ -0,0 +1,4 @@
+export interface ChatBotMessage {
+  role: 'system' | 'assistant' | 'user';
```

Comment:
Maybe we want to expose an enum with these values, so it is easier to construct messages without having to remember the exact strings:
```ts
export enum ChatbotRole {
  System = "system",
  Assistant = "assistant",
  User = "user
}
```
And you can get the union of enumerated values like:
```suggestion
  role: `${ChatbotRole}` | ChatbotRole;
```

Reply:
That's a good idea !